external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/MsSqlServerIntegrationSuite.scala

```diff
@@ -59,7 +59,7 @@ class MsSqlServerIntegrationSuite extends DockerJDBCIntegrationSuite {
       """
         |INSERT INTO numbers VALUES (
         |0,
-        |255, 32767, 2147483647, 9223372036854775807,
+        |127, 32767, 2147483647, 9223372036854775807,
```

Comment:
Wait, how was this possible before? Do we cover unsigned cases too?

Reply:
Yes, all test cases including signed and unsigned are covered now for ByteType and ShortType. Refer to test("SPARK-29644: Write tables with ShortType") and  test("SPARK-29644: Write tables with ByteType") in JDBCWriteSuite.scala and MsSQLServerIntegrationSuite.scala.

Earlier it all worked as everything was treated as an integer. Every test cases treated ByteType and ShortType as integers. These test are corrected now.

Reply:
I meant 

1. This TINYINT seems able to contain unsigned values (https://dev.mysql.com/doc/refman/8.0/en/integer-types.html) up to 255. How do we handle?
2. Previously the value given here was `255` for `TINYINT` which is performed via MySQL if I am not mistaken. How was this possible without UNSIGNED keyword?

Reply:
@HyukjinKwon .
It seems that we need to revert https://github.com/apache/spark/pull/23400 with the same reason. How do you think about that?